src/js/components/widgets/widget-color/widget-color-box.react.js

```diff
@@ -36,7 +36,7 @@ class WidgetColorBox extends React.Component {
                         </div>
                     </div>
                     <div className='color'>
-                        <div className='activeColor' style={{ backgroundColor: 'rgba(' + this.props.rgb.r + ', ' + this.props.rgb.g + ', ' + this.props.rgb.b + ', ' + this.props.rgb.a + ')' }}/>
+                        <div className='activeColor' style={{ backgroundColor: 'rgba(' + this.props.color.r + ', ' + this.props.color.g + ', ' + this.props.color.b + ', ' + this.props.color.a + ')' }}/>
```

Comment:
In addition to namespacing class names in this component, class names should be lowercase with hyphens (e.g. `active-class`) instead of camelCased.